src/starcheck.pl

```diff
@@ -678,10 +678,16 @@ sub json_obsids{
         my $att_ok = make_pcad_attitude_check_report(
             $backstop, $or_file, $mm_file, $char_file, $att_report);
         if ($att_ok){
-            $out .= "<A HREF=\"${att_report}\">[OK] Coordinates as expected.</A>\n\n";
+            $out .= "<A HREF=\"${att_report}\">[OK] Coordinates as expected.</A>\n";
         }
         else{
-            $out .= "<A HREF=\"${att_report}\">[NOT OK] Coordinate mismatch or error.</A>\n\n";
+            $out .= "<A HREF=\"${att_report}\">[NOT OK] Coordinate mismatch or error.</A>\n";
+        }
+        if (check_characteristics_date($char_file, $date[0])){
+            $out .= "<A HREF=\"${att_report}\">[OK] Characteristics file newer than 30 days</A>\n\n";
```

Comment:
Nah, that wouldn't work in all cases where starcheck is called from outside of the products directory.  So we'd need to copy the characteristics into $STARCHECK and I suppose that would increase the risk that they'd get copied somewhere they shouldn't.